titan-server/src/bin/main.rs

```diff
@@ -8,30 +8,27 @@ extern crate diesel;
 extern crate regex;
 extern crate bcrypt;
 extern crate frank_jwt;
-extern crate rocket_cors;
 
 use libtitan::routes;
 use libtitan::db;
 use libtitan::config;
-use libtitan::cors_fairing;
 use libtitan::accounts;
 use libtitan::organizations;
 
 fn main() {
     rocket::ignite()
         .attach(db::TitanPrimary::fairing())
         .attach(db::UnksoMainForums::fairing())
-        .attach(cors_fairing::cors())
```

Comment:
If we're no longer using `rocket_cors`, it should be removed from the apps dependencies in `Cargo.toml`.

Reply:
Yeah, I wasn't sure if I had completely removed all references to it. In fact, there are some in the library code that I need to deal with.